example/lib/main.dart

```diff
@@ -223,7 +223,9 @@ class ChannelPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: ChannelHeader(),
+      appBar: ChannelHeader(
+        showTypingIndicator: false,
+      ),
```

Comment:
Is this correct as per Figma designs?

Reply:
yep, I checked with Francesco
We support typing indicators in channelheader, but in the sample app we show them at the bottom of the messagelistview, you can check on figma

Reply:
Cool, LGTM then.